scripts/kconfig/hardenconfig.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from kconfiglib import Symbol, standard_kconfig
+
+import csv
+import os
```

Comment:
pylint3 complains that these imports should be before the "from <> import <>" line above. It is a PEP8 style guide (if we intend to follow that)